tensorbay/opendataset/DAVIS2017/loader.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+import os
+from itertools import product
+from typing import Callable, Dict, Iterator
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import InstanceMask, SemanticMask
+from tensorbay.opendataset._utility import glob
+
+_RESOLUTIONS = ("480p", "Full-Resolution")
+_SEMI_SUPERVISED_DATASET_NAME = "DAVIS2017SemiSupervised"
+_UNSUPERVISED_DATASET_NAME = "DAVIS2017Unsupervised"
+
+
+def DAVIS2017SemiSupervised(path: str) -> Dataset:
+    """Dataloader of `DAVIS2017SemiSupervised`_ open dataset.
+
+    .. _DAVIS2017SemiSupervised: https://davischallenge.org/davis2017/code.html
+
+    The file structure should be like::
+
+        <path>
+            Annotations/
+                480p/
+                    aerobatics/
+                        00000.png
+                    bear/
+                    ...
+                Full-Resolution/
+                    aerobatics/
+                        00000.png
+                    bear/
+                    ...
+            Annotations_semantics/
+                480p/
+                    aerobatics/
+                        00000.png
+                    bear/
+                    ...
+                Full-Resolution/
+                    aerobatics/
+                        00000.png
+                    bear/
+                    ...
+            JPEGImages/
+                480p/
+                    aerobatics/
+                        00000.jpg
+                        00001.jpg
+                        ...
+                    bear/
+                    ...
+                Full-Resolution/
+                    aerobatics/
+                        00000.jpg
+                        00001.jpg
+                        ...
+                    bear/
+                    ...
+            ImageSets/
+                2017/
+                    test-challenge.txt
+                    test-dev.txt
+                    train.txt
+                    val.txt
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class: `~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+    dataset = Dataset(_SEMI_SUPERVISED_DATASET_NAME)
+    dataset.notes.is_continuous = True
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog_semi_supervised.json"))
+
+    image_set_types: Dict[str, Callable[..., Data]] = {
```

Comment:
better to use value to name the dict rather than key
`image_set_types` -> `data_getters`?